app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
Kindly modify your Login page view as per the provided wire frame design including styling. This is also a part of the requirements of this milestone.
| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/195975776-547925aa-844b-4777-914b-eb7650c8d4fc.png) | ![image](https://user-images.githubusercontent.com/33268927/195975804-5d8b7513-fe28-446c-812f-58d822dcd203.png) |